libminifi/include/core/controller/StandardControllerServiceProvider.h

```diff
@@ -111,10 +111,10 @@ class StandardControllerServiceProvider : public ControllerServiceProvider, publ
     for (auto service : controller_map_->getAllControllerServices()) {
 
       if (service->canEnable()) {
-        logger_->log_info("Enabling %s", service->getName());
```

Comment:
Ditto on the comments about enabling/starting something be critical for tier 1 to support.